site/src/components/AvatarData/AvatarData.tsx

```diff
@@ -48,5 +48,6 @@ const useStyles = makeStyles((theme) => ({
   },
   avatar: {
     marginRight: theme.spacing(1.5),
+    background: "hsl(219, 8%, 52%)",
```

Comment:
Do you think we could have it as a `palette` object like `theme.palette.paper.dark`? Or something like that?